coord/src/info_command.c

```diff
@@ -49,9 +49,13 @@ static InfoFieldSpec toplevelSpecs_g[] = {
     {.name = "cleaning", .type = InfoField_WholeSum}};
 
 static InfoFieldSpec gcSpecs[] = {
-    {.name = "current_hz", .type = InfoField_DoubleAverage},
     {.name = "bytes_collected", .type = InfoField_WholeSum},
-    {.name = "effectiv_cycles_rate", .type = InfoField_DoubleAverage}};
+    {.name = "total_ms_run", .type = InfoField_WholeSum},
+    {.name = "total_cycles", .type = InfoField_WholeSum},
+    {.name = "average_cycle_time_ms", .type = InfoField_DoubleAverage},
```

Comment:
@GuyAv46 
Small question about this field - the original field is already an average.
After the aggregation it is going to be an average of averages.
Wondering if we are better off having a special case for this field or if keeping it as a weighted average is good enough.

Reply:
Since the number of cycles on each chard should be equal (or relatively close), the average of averages should be close to the actual average.
I think this is OK. @oshadmi WDYT?

Reply:
@GuyAv46 Why not compute the accurate average from `total_cycles` and `total_ms_run`?

Reply:
I missed that we have this data. If it's not too complicated lets do it as a spacial case

Reply:
@GuyAv46 
Tried to implement the requested behavior.
Let me know what you think of the change.